kubernetes/architecture.json

```diff
@@ -14,40 +11,6 @@
   "discoveryQuery": [
     "_exists_:kubernetes_cluster"
   ],
-  "propertyColumns": [
```

Comment:
I can see why `listColumns` aren't needed in architecture views but is `propertyColumns` not used for the properties tab when you click on a node?